crates/bitwarden-exporters/src/lib.rs

```diff
@@ -132,3 +134,9 @@ pub struct Identity {
     pub passport_number: Option<String>,
     pub license_number: Option<String>,
 }
+
+pub struct SshKey {
+    pub private_key: Option<String>,
```

Comment:
Can we document the format we use for private keys? For passkeys we use PKCS8 Der, https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto/importKey#pkcs_8. I would suggest making sure we use the same here, since CXP will also use DER.

Reply:
So far I've favored OPENSSH for SSH keys because
1.) Ssh clients (in general) don't support ed25519 in pkcs8 format, and
2.) the `ssh-key` crate does not support ssh keys in pkcs8. 

If we save in pkcs#8, this would mean that we either do import/export in pkcs#8, which would block users from using their ed25519 keys in openssh clients directly, or import/export would need to wait for sdk support in e.g. web.

Additionally, we would need to write functions to convert from openssh to pkcs8 when generating keys with ssh-keys, similar to https://github.com/bitwarden/clients/pull/11048

Reply:
Added documentation for that we export in openssh-privatekey format (and for the other fields).